src/libraries/JANA/Services/JParameterManager.h

```diff
@@ -284,8 +285,8 @@ JParameter* JParameterManager::SetDefaultParameter(std::string name, T& val, std
 
     // Always put val through the stringification/parsing cycle to be consistent with
     // values passed in from config file, accesses from other threads
-    Parse(param->GetValue(),t);
-    val = t;
+    Parse(param->GetValue(),t1);
+    val = t1;
```

Comment:
This should fix the bug, but I think it misses the root cause, which is that the vector version of Parse() is appending to t without clearing it first.

Reply:
We might run into other bugs with `Parse<std::string, std::vector<T> val>()` down the road if we don't clear `val`